sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/export/IntervalMetricReader.java

```diff
@@ -37,7 +38,10 @@ public void shutdown() {
     scheduler.shutdown();
     try {
       scheduler.awaitTermination(5, TimeUnit.SECONDS);
-      exporter.run();
+      CompletableResultCode result = exporter.forceFlush();
+      if (result != null) {
```

Comment:
Can we make this return value non-null, so we don't have to do this check? Also, I'd like to have the `IntervalMetricReader.shutdown()` method also return `CompleteableResultCode` as well.

Reply:
Thanks. will make the change.